src/main/java/com/cinema/model/dto/movie/MovieDtoMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.cinema.model.dto.movie;
+
+import com.cinema.model.Movie;
+import com.cinema.service.MovieService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieDtoMapper {
+    private final MovieService movieService;
+
+    public MovieDtoMapper(MovieService movieService) {
+        this.movieService = movieService;
+    }
+
+    public Movie getMovie(MovieRequestDto dto) {
+        Movie movie = new Movie();
+        movie.setTitle(dto.getTitle());
+        return movie;
```

Comment:
What if we receive the full info about the movie from the FrontEnd-table consisting of all the movies?
Example : 
id______title______description
1 _____ Test 1 ____ Desc 1
2 _____ Test 2 ____ Desc 2

In my opinion, we should create the ready-to-work object or get from DB by the title, in such a case

Reply:
I've added ```description``` field and setting it in mapper. But getting movie from DB is also good idea